src/server/src/main/docker/entrypoint.sh

```diff
@@ -2,9 +2,9 @@
 
 if [ "$1" = 'cassandra-reaper' ]; then
     set -x
-    su-exec reaper /usr/local/bin/append-persistence.sh
-    exec su-exec reaper java -jar ${JAVA_OPTS} \
-        /usr/local/lib/cassandra-reaper.jar server /etc/cassandra-reaper.yml
+    su-exec reaper /usr/local/bin/configure-persistence.sh
```

Comment:
Any idea on why some are `su-exec` and one is `exec su-exec`? Does the former fork the process or something?

Reply:
Unsure to be honest.